code/WorkInProgress/zamu_mail.dm

```diff
@@ -628,10 +756,52 @@ var/global/mail_types_by_job = list(
 		)
 	)
 
+// =========================================================================
+// Various items given out based on player trait choices, to make it more personal
+var/global/list/mail_types_by_trait = list(
+	"petasusaphilic" = filtered_concrete_typesof(/obj/item/clothing/head, /proc/filter_trait_hats),
+	"pawnstar" = trinket_safelist,
+	"petperson" = filtered_concrete_typesof(/mob/living/critter/small_animal/, GLOBAL_PROC_REF(filter_carrier_pets))\
```

Comment:
Seems odd to get a whole new pet every time, would maybe make more sense to get dog biscuits, kibble, fish etc?